Header header
uint8[] data

#0 UWB 1 WiFI 2 All
uint8 send_method